arch/renesas/src/rx65n/rx65n_irq.c

```diff
@@ -426,6 +426,74 @@ void up_disable_irq(int irq)
 
 #endif
 #endif
+
+#ifdef CONFIG_RX65N_RSPI0
+  if (irq == RX65N_SPRI0_IRQ)
+    {
+      ICU.IER[04].BIT.IEN6 = 0;
```

Comment:
Avoid using number preceding of '0'! A number preceeding of 0 is an octal number, this way 08 and 09 are illegal.